indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -77,10 +77,13 @@ def _checkParams(self, params):
     def _process(self):
         query = (Abstract
                  .query.with_parent(self.event_new)
+                 .filter(Abstract.state.notin_({AbstractState.duplicate, AbstractState.merged}))
```

Comment:
We can't have abstract A being a duplicate of abstract B when abstract B is a duplicate of abstract C?